### PR TITLE
Add username to contributors

### PR DESCRIPTION
**Pull request by Fehings**

### CODE REVIEW

**Comment by rozzerspb** [comment]
I agree

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 - Mtbu
 - Missive
 - Kiguli
-- rozzerspb
\ No newline at end of file
+- rozzerspb
+- fehings
```
Comment (Fehings) [inline CONTRIBUTORS.md:5]: I don't understand

**Review by rozzerspb — APPROVED** [review]
love it

**Comment by Kiguli** [comment]
Warning: this might delete main... too late.

**Pull request merged**